docs/auth_api.md

```diff
@@ -219,3 +219,35 @@ If the access token is no longer valid, you will get a response with HTTP status
 [oauth2-spec]: https://tools.ietf.org/html/rfc6749
 [indieauth-spec]: https://indieauth.spec.indieweb.org/
 [indieauth-clients]: https://indieauth.spec.indieweb.org/#client-identifier
+
+## Signed paths
+
+Sometimes you want a user to make a GET request to Home Assistant to download data. In this case the normal auth system won't do, as we can't link the user to an API with the auth header attached to it. In that case, a signed path can help.
+
+A signed path is a normal path on our server, like `/api/states`, but with an attached secure authentication signature. The user is able to navigate to this path and will be authorised as the access token that created the signed path. Signed paths can be created via the websocket connection and are meant to be shortlived. The default expiration is 30 seconds.
```

Comment:
Authorized

Reply:
that's debatable 😉 https://english.stackexchange.com/questions/282621/authorization-vs-authorisation-im-in-some-real-dilemma

Reply:
Most answer in the link are for z 🤔 but I guess since you are in Europe `s` is okay haha

Reply:
In this instance I dont think it will matter so merging